WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
I meant the scrolling to # today.

But the header is not floating as well.
Em 08/02/2016 17:31, "Jeremy Fleischman" <notifications@github.com>
escreveu:

> In WcaOnRails/app/views/competitions/_index_form.html.erb
> <https://github.com/cubing/worldcubeassociation.org/pull/319#discussion_r52195676>
> :
>
> > @@ -0,0 +1,27 @@
> > +<div class="form-group">
>
> Can we get the buttons to be side by side on mobile? This feels like a big
> waste of space currently.
>
> By "anchor", do you mean the floatThead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/319/files#r52195676>
> .
>